cerberus-web/src/main/java/com/nike/cerberus/util/CiphertextUtils.java

```diff
@@ -55,13 +52,7 @@ public static ParsedCiphertext parse(byte[] ciphertext) {
     List<String> cmkArns = Lists.newArrayList();
     for (KeyBlob keyBlob : parsedCiphertext.getEncryptedKeyBlobs()) {
       if (KMS_PROVIDER_ID.equals(keyBlob.getProviderId())) {
-        try {
-          cmkArns.add(new String(keyBlob.getProviderInformation(), "UTF-8"));
-        } catch (UnsupportedEncodingException e) {
-          LoggerFactory.getLogger("com.nike.cerberus.util.CiphertextUtils")
-              .error("Failed to create new string from key blob provider information", e);
-          throw new ApiException(DefaultApiError.INTERNAL_SERVER_ERROR);
-        }
+        cmkArns.add(new String(keyBlob.getProviderInformation(), StandardCharsets.UTF_8));
```

Comment:
if we pass StandardCharsets object instead of charset name as string to String constructor we could eliminate catching compiletime UnsupportedEncodingException